packages/front-end/components/Features/SDKConnections/SDKLanguageSelector.tsx

```diff
@@ -14,6 +14,7 @@ const tabs: Record<LanguageFilter, string> = {
   server: "Server",
   mobile: "Mobile",
   edge: "Edge",
+  essential: "Essential",
```

Comment:
Probably not necessary because we can use `limitLanguages` with `SDKLanguageSelector`. Going to see if I can remove this new `LanguageFilter`